tests/pass-dep/shims/libc-fs.rs

```diff
@@ -193,3 +258,20 @@ fn test_posix_mkstemp() {
         assert_eq!(e.kind(), std::io::ErrorKind::InvalidInput);
     }
 }
+
+// Trait implemented for test_ftruncate to convert i64 to the provided type
+trait Length: Sized {
```

Comment:
Why are you defining a new trait for this? The standard library has `From<i32>` that you should be able to directly use.

Reply:
Yes, ``From<i32>`` works. I haven't found a correct way to cast ``i64`` to ``i32`` so I resorted to implementing traits. Thanks for pointing this out.